### ISSUE TITLE
No response from machine

### ISSUE THREAD

**Issue by behold81** [duckwc/ECAMpy#4]
Using a primadonna soul and i have the python script working but I do not get any response back from the webservice. I am on the same wifi network as the machine but it jut continually runs get status and no actual response is returned. 

Thoughts?

Secondly have you integrated this in to Home assistant at all?

**Comment by jonshorter** [comment]
Did you notice that the SRV_IP should be the IP address of your PC, not the coffee machine? The coffee machine IP is further down (DEV_IP) - that's a mistake I made at first, so thought I'd offer it up as a suggestion to check. Also, try turning off your firewall to test.

**Comment by behold81** [comment]
That worked. now to work out how to make this HA integrated so my machine is ready when I get up in the morning!

**Comment by jonshorter** [comment]
Yeah I think this project has legs... we just need to put some time into it. Unfortunately I don't have an abundance of that right now, but if someone could create the integration framework, I'm down for testing, tweaking and a bit of development.

**Comment by oksibutch** [comment]
Did anyone go further and make the coffee machine turn on through HA?

**Comment by miditkl** [comment]
> Did anyone go further and make the coffee machine turn on through HA?

Good news: [cremalink-ha](https://github.com/miditkl/cremalink-ha). It is working (for me). I also made some automations/scenes. Machine was responsive 24/7.
